hil-test/tests/crc.rs

```diff
@@ -0,0 +1,68 @@
+//! CRC and MD5 Tests
+
+//% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32h2 esp32s2 esp32s3
+
+#![no_std]
+#![no_main]
+
+use core::ops::Deref;
+
+use defmt_rtt as _;
+use esp_backtrace as _;
+use esp_hal::rom::{crc, md5};
+
+#[embedded_test::tests]
+mod tests {
+    use defmt::assert_eq;
+
+    use super::*;
+
+    #[init]
+    fn init() {}
```

Comment:
Just out of curiosity, does `embedded_test` require an `init` function, or are we just including it to be consistent with the other tests? It's not a problem at all as-is, of course.